<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagWidget</class>
 <widget class="QWidget" name="TagWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>749</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="edtTags"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Assign:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbAssignTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To Fandom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFandom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkTagIncludingCrosses">
       <property name="text">
        <string>Crosses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAssignTagToFandom">
       <property name="text">
        <string>Do It</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbDeleteTag">
       <property name="text">
        <string>Delete Tag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTag"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddTag">
       <property name="text">
        <string>Add Tag</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbTagExport">
       <property name="text">
        <string>Export Tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbTagImport">
       <property name="text">
        <string>Import Tags</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>edtTags</zorder>
  <zorder></zorder>
  <zorder>pbTagExport</zorder>
  <zorder></zorder>
  <zorder>pbTagImport</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
